app/controllers/organizations/faqs_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Organizations::FaqsController < Organizations::BaseController
+  layout "dashboard"
+  before_action :context_authorize!, only: %i[index new create]
```

Comment:
Any reason to leave out Update and Destroy?

Reply:
I'm still trying to fully wrap my head around action policy, however I reviewed default_pet_task to get a better understanding and this is how it is set up there.

`context_authorize!` calls `authorize! DefaultPetTask,
      context: {organization: Current.organization}`

For show edit update destroy  uses `set_faq` which calls `authorize! @faq`